package.json

```diff
@@ -28,6 +28,7 @@
     "express-pino-logger": "^6.0.0",
     "helmet": "4.4.1",
     "http-errors": "^1.8.0",
+    "ioredis-mock": "^5.4.1",
```

Comment:
This shouldn't be part of this PR

Reply:
oh my, thanks for the heads up. it's removed now

Reply:
But now the tests are failing because it was removed haha

Reply:
Yeah lol, I just fixed it along with fixing the misspelt function names